src/components/Explore/utils.ts

```diff
@@ -429,42 +420,9 @@ export function getSubtitle(
     : `${metricName} ${textPer100k} in ${getLocationNames(regions)}`;
 }
 
-/**
- * Returns a list of locations for the autocomplete component. We try to show
- * the most relevant options first. For a state, we show states, counties in
- * the state and then other counties. For a county, we show counties in the
- * state, then states and then other counties.
- */
-export function getAutocompleteLocations(locationFips: string) {
-  const states = regions.states;
-  const allCounties = regions.counties;
-
+export function getExploreAutocompleteLocations(locationFips: string) {
   const currentLocation = regions.findByFipsCode(locationFips)!;
-  const stateCode = getStateCode(currentLocation);
-
-  const [stateCounties, otherCounties] = partition(
-    allCounties,
-    county => county.stateCode === stateCode,
-  );
-
-  const sortedStates = sortBy(states, location => location.name);
-  const sortedStateCounties = sortBy(stateCounties, location => location.name);
-  const sortedOtherCounties = sortBy(otherCounties, location => location.name);
-
-  // TODO(michael): Where should aggregations go in the list?
-  return currentLocation.regionType === RegionType.STATE
-    ? [
-        ...regions.customAreas,
-        ...sortedStates,
-        ...sortedStateCounties,
-        ...sortedOtherCounties,
-      ]
-    : [
-        ...regions.customAreas,
-        ...sortedStateCounties,
-        ...sortedStates,
-        ...sortedOtherCounties,
-      ];
+  return currentLocation ? getSearchAutocompleteLocations(currentLocation) : [];
```

Comment:
This feels a bit like it's tying two different areas together - thought on making a generic `getAutoCompleteLocations(region?: Region)` that both use?

Reply:
Yeah, I can move the function to a more central place, as the logic is the same for both